source/agora/consensus/data/Transaction.d

```diff
@@ -124,24 +131,32 @@ public struct Input
     /// A signature that should be verified using the `previous[index].address` public key
     public Signature signature;
 
+    /// The UTXO this `Input` references must be at least `unlock_age` older
+    /// than the block height at which the spending transaction wants to be
+    /// included in the block. Use for implementing relative time locks.
+    public uint unlock_age = 0;
```

Comment:
Oh. Stoa needs to respond to this change.

Reply:
Yes, to both changes.